locales/templates/app.pot

```diff
@@ -1132,11 +1132,48 @@ msgctxt "preference_autocomplete_duplicate_url_error"
 msgid "URL already exists"
 msgstr ""
 
-#. Find in page
+#. Label for the Find in page button
 msgctxt "find_in_page"
 msgid "Find in Page"
 msgstr ""
 
+#. Watermark/Hint for the find in page input field
+msgctxt "find_in_page_input"
+msgid "Find in page"
+msgstr ""
+
+#. String to show the number of results found in the page and the
+#. position the user is at. The first argument is the position, the second
+#. argument is the total
+#, c-format
+msgctxt "find_in_page_result"
+msgid "%1$d/%2$d"
+msgstr ""
+
+#. String to be read by the accessibility service
```

Comment:
I think this needs to be explained better because as a stand alone comment it's hard to understand what we're talkign about here (what does each placeholder stand for). Depending on what language it's translated in, this might affect the position of the placeholders.